src/components/details/TimestampWrapper.tsx

```diff
@@ -0,0 +1,82 @@
+import * as _ from 'lodash';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import {
+  K8sModel,
+  K8sResourceCommon,
+  K8sResourceKindReference,
+  Timestamp,
+  useK8sModel,
+  useK8sWatchResources,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { getReference } from '@openshift-console/dynamic-plugin-sdk/lib/utils/k8s/k8s-ref';
```

Comment:
You know what is the newer API that we should use?  This is marked deprecated

Reply:
Yes, but K8sGroupVersionKind is not provided by the SDK now, we will need to revisit this issue later.

Reply:
Hey Yi, could open a Jira story to track this?

Reply:
@keithchong  Sure. We don't know when this will happen so I assume it's not in the dynamic plugin epic?

Reply:
Jira ticket created: https://issues.redhat.com/browse/GITOPS-2316